### ISSUE TITLE
Add support for Yarn 2 Pnp

### ISSUE THREAD

**Issue by alexkuc** [firsttris/vscode-jest-runner#104]
This plugin is awesome but it is missing support for Yarn 2 Pnp. I have created a PR #103 for this. Let me know what you think (PR itself contains technical details).

**Comment by grrizzly** [comment]
I was able to glean the settings changes from @alexkuc 's PR to form a temporary workaround.

Modify `.vscode/settings.json` with the following options to get PnP support:

```
"jestrunner.jestCommand": "yarn jest",
  "jestrunner.jestPath": "`yarn bin jest`",
  "jestrunner.debugOptions": {
    "runtimeArgs": ["--require", "./.pnp.js"]
  }
```

Hopefully the PR gets merged quickly; posting just in case someone needs it in the meantime.

**Comment by firsttris** [comment]
available in version 0.4.24
